helm/templates/dist.yml

```diff
@@ -0,0 +1,156 @@
+{{ if .Values.dist }}
+{{ range $dist := .Values.dist.pops }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "dist.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "name" $ }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    run: {{ include "dist.fullname" . }}
+spec:
+  type: ClusterIP
+  ports:
+  - port: 5353
+    targetPort: data
+    protocol: TCP
+    name: data
+  - port: 3300
+    targetPort: http-config
+    protocol: TCP
+    name: http-config
+  - port: 9090
```

Comment:
This shouldn't be exposed

Reply:
Does it need to be exposed on the pod? What's it used for?